dist/documentation/themes/dark.d.ts

```diff
@@ -564,4 +564,37 @@ export const shadow : {base:{
             name: string,
             hex: string,
             description: string
+          }};
+export const accent : {forestTextBase:{
+            name: string,
+            hex: string,
+            
+          },forestBackgroundMuted:{
+            name: string,
+            hex: string,
+            
+          },forestBackgroundMutedHover:{
+            name: string,
+            hex: string,
+            
+          },forestBackgroundMutedActive:{
+            name: string,
+            hex: string,
+            
+          },sandTextBase:{
+            name: string,
+            hex: string,
+            
+          },sandBackgroundMuted:{
+            name: string,
+            hex: string,
+            
+          },sandBackgroundMutedHover:{
+            name: string,
+            hex: string,
+            
+          },sandBackgroundMutedActive:{
```

Comment:
i wonder if we need an accent transform 🫠, so this is nested more. `accent.sand.backgroundMutedActive`


or alternatively, remove the `accent` level, but we prob wanna keep that

Reply:
Yeah, I think we should probably nest one more, will likely be too many options, especially when we are wanting to possibly add more for things like `ApplePay` and such that have specific brand colors

Reply:
Unless we foresee these accent colours being used immediately, let's hold off on transforming them for now. It's a bit more challenging than expected, and isn't priority compared to getting DS4 out. I'll make a ticket for the backlog to tackle this

Reply:
Sounds good @alexandra-lim, agree on prioritizing getting something initial out for DS4 release